src/_P076_HLW8012.ino

```diff
@@ -743,9 +754,9 @@ const __FlashStringHelper* p076_getQueryString(uint8_t query) {
   {
     case 0: return F("Voltage_V");
     case 1: return F("Current_A");
-    case 2: return F("Active Power_W");
-    case 3: return F("Reactive Power_VAR");
-    case 4: return F("Apparent Power_VA");
+    case 2: return F("Active_Power_W");
+    case 3: return F("Reactive_Power_VAR");
+    case 4: return F("Apparent_Power_VA");
     case 5: return F("Power_Factor_cosphi");
     case 6: return F("Energy_Ws");
```

Comment:
See what I did for the P082_GPS in `Plugin_082_valuename`
There I have the display string and a task value string.
The display string is shown in the combobox when selecting the output value.
The non-display string is the shorter taskvalue name.
So maybe that's also more useful here as you would otherwise get really long strings for `[taskname#taskvaluename]` which may be an issue when showing on some display as most line fields for display plugins have a limited string length.
Also it will probably mess up the layout on the "Devices" tab.

Reply:
You're right, that wouldn't look good on the overview. I have adapted the strings based on the P082 plugin.